### ISSUE TITLE
Some pages are not in this repo

### ISSUE THREAD

**Issue by isomiki** [metaDAOproject/meta-dao-docs#6]
There are some pages on the public docs website which are not in this repo:
- The introduction: https://docs.metadao.fi/
- Everything under "using the platform", e.g. https://docs.metadao.fi/using-the-platform/creating-a-dao
- ELI5 page: https://docs.metadao.fi/futarchy/eli5
- Media kit pages, e.g. https://docs.metadao.fi/media-kit/metadao

There might be some others.

**Comment by R-K-H** [comment]
See https://github.com/metaDAOproject/futarchy/tree/develop/docs
